src/page-building/landingpage/blog.tsx

```diff
@@ -0,0 +1,47 @@
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import { TeaserGrid } from '../../components/teasers/teaser-grid';
+import { Teaser } from '../../components/teasers/teaser';
+import React from 'react';
+import { HomePageHeaderBlock } from './support';
+import { BlogPostTeaser } from '../../@types/blog';
+import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+import { useLocalFormat } from '../../new-components/i18n-helpers';
+
+interface BlogProps {
+  posts: BlogPostTeaser[];
+}
+
+export const Blog = ({ posts }: BlogProps) => {
+  const { t } = useTranslation();
+  const dateFormatter = useLocalFormat('dd. MMMM yyyy');
+
+  return (
+    <>
+      <HomePageHeaderBlock
+        topline={t('main.blog.kicker')}
+        headline={t('main.blog.title')}
+        large={true}
+      >
+        {t('main.blog.text')}
+      </HomePageHeaderBlock>
+
+      <TeaserGrid>
+        {posts.map((item) => {
+          const image = getImage(item.frontmatter.featuredImage)!;
+
+          return (
+            <Teaser
+              key={item.id}
+              title={item.frontmatter.title}
+              linkTo={item.frontmatter.path}
+              dateFormatted={dateFormatter(item.frontmatter.date)}
+              cover={image && <GatsbyImage alt="" image={image} />}
+            >
+              {item.frontmatter.teaserText}
+            </Teaser>
+          );
+        })}
+      </TeaserGrid>
+    </>
+  );
+};
```

Comment:
shiny new blog section for the landingpage. I don't like how the teaser render the actual blog content.
the design does not work for us. 

+ We don't need the kicker
+ The kicker conflicts with the timestamp easily (same line)
+ I don't have a place to easily show the author which we might want to do.

![Screen Shot 2021-12-10 at 14 13 41](https://user-images.githubusercontent.com/1701755/145579572-51dbba0a-ef66-4dcd-bf47-c27283f8a3f3.png)